src/sidebars/star/star.less

```diff
@@ -15,34 +19,40 @@
   outline: 0;
   display: block;
   .confirmation-modal {
+    &.flipped{
+      box-shadow: -2px -2px 2px 0px @dark-background;
+    }
+    box-shadow: -2px 2px 2px 0px @dark-background;
```

Comment:
Looking at it the blur might always be the same, so it would just be:

`.box-shadow(-2px -2px, @dark-background)`